frontend/packages/sdk/src/lib/types.ts

```diff
@@ -31,6 +31,7 @@ export const types: RegistryTypes = {
         payload: "Vec<u8>",
         deposit: "u128",
         transfer: "u128",
+        estimating: "bool",
```

Comment:
Please set this field to true when estimating gas fee on the UI. @Leechael

Reply:
In order to implement this change, we will need to ensure that it is synced across all testnets.